web/client/themes/default/less/map-toolbar.less

```diff
@@ -27,6 +27,15 @@
     margin-bottom: 35px;
     margin-right: 5px;
 }
+.mapToolbarHorizontal {
```

Comment:
if you pass a style object this is no longer needed

Reply:
Done passing style object as a prop